fdbrpc/include/fdbrpc/LoadBalance.actor.h

```diff
@@ -365,7 +372,7 @@ Future<Void> replicaComparison(Req req,
 							        src.get(),
 							        f.get().get().get())) { // re-use TSS compare logic to compare the replicas
 								numMismatch++;
-								TraceEvent mismatchEvent(SevError, LB_mismatchTraceName(req, REPLICA_COMPARISON));
+								TraceEvent mismatchEvent(SevWarnAlways, LB_mismatchTraceName(req, REPLICA_COMPARISON));
```

Comment:
The type should be "SevError" if (requiredReplicas == ALL_REPLICAS)?

Reply:
Type "SevError" is used above on "throw unreachable_storage_replica();" if (requiredReplicas == ALL_REPLICAS). The type needs to be consistent in both cases.

Reply:
Done.